client/src/App.jsx

```diff
@@ -18,22 +25,31 @@ function App() {
 				<Router>
 					<Routes>
 						<Route
-							path='/'
-							element={<LandingPage />}
-						/>
+							element={
+								<>
+									<Navbar></Navbar>
+									<Outlet></Outlet>
+									<Footer></Footer>
+								</>
+							}>
+							<Route
+								path="/"
+								element={<LandingPage />}
+							/>
+						</Route>
 						<Route
-							path='/login'
+							path="/login"
```

Comment:
For some reason I'm redirected to the landing page when I try to navigate to /login. 🤔 Every other existing route works though.

Reply:
You are correct, some changes were made to the login page component that seems to have made it auto nav to the landing page even without a logged-in user. I don't believe this is an issue with the App.jsx route changes. I will report this in the Bug Thread.